assets/src/dashboard/components/toggleButtonGroup/index.js

```diff
@@ -82,6 +82,12 @@ ToggleButton.propTypes = {
   isActive: PropTypes.bool,
 };
 
+export const ToggleButtonGroupContainer = styled.div`
+  display: block;
+  flex: 1;
+  height: 65%;
```

Comment:
I feel like this height is specific to the pageHeader context, would this make more sense in the shared components of views so it can be referenced for saved templates that has the same structure but be in a spot that's specific to the layout as opposed to just basic components?

Reply:
Good idea. I think we could name this `HeaderToggleButtonGroupContainer` to be very specific